docs/deployment/feature_gates.md

```diff
@@ -108,6 +106,9 @@ The following tables are a summary of the feature gates that you can set on diff
 | ShootSARotation                              | `true`  | `Beta`       | `1.51` | `1.56` |
 | ShootSARotation                              | `true`  | `GA`         | `1.57` | `1.59` |
 | ShootSARotation                              |         | `Removed`    | `1.60` |        |
+| ReversedVPN                                  | `false` | `Alpha`      | `1.22` | `1.41` |
+| ReversedVPN                                  | `true`  | `Beta`       | `1.42` | `1.61` |
+| ReversedVPN                                  | `true`  | `GA`         | `1.61` |        |
```

Comment:
Should this not be 1.62 as we are currently in the 1.62 development cycle?

@rfranzke Feel free to clarify.

Reply:
I doubt that this will make it into 1.62 - let's rather go for 1.63 directly